deepchem/models/torch_models/ferminet.py

```diff
@@ -274,13 +292,14 @@ def __init__(self,
         self.model = Ferminet(nucl,
                               spin=(self.up_spin, self.down_spin),
                               nuclear_charge=torch.Tensor(charge),
-                              batch_size=self.batch_no)
+                              batch_size=self.batch_no).float()
 
         self.molecule: ElectronSampler = ElectronSampler(
             batch_no=self.batch_no,
             central_value=self.nucleon_pos,
             seed=self.seed,
-            f=lambda x: test_f(x),  # Will be replaced in successive PR
+            f=lambda x: self.random_walk(x
```

Comment:
changing the sampling function to the random walk function which returns the log wavefucntion probability